### PR TITLE
Feat/common custom rules

### PR DESCRIPTION
**Pull request by marinofelipe**
### Why?
To create custom rules Pod

### Changes
- Set another .podspec for common custom rules
- Add `greater than or equal filed` custom rule

### Tests 
- Add negative and positive tests for the custom rule

### [Issue](https://github.com/grupozap/squad-canalpro/issues/1177)

### CODE REVIEW

File: LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+//  Copyright © 2019 Zap SA Internet. All rights reserved.
+//
+
+import Foundation
+import Livr
+
+struct CustomRules {
+    
+    struct GreaterThanOrEqualToField: LivrRule, PreDefinedRule, RuleThatCompareFields, CustomRule {
+        var name: String = "greater_eq_than_field"
```
Comment (renatomedina) [inline LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift]: private? What do you think?

File: LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        static var name = "greater_eq_than_field"
```
Comment (renatomedina) [inline LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift]: Why do you create two variables names with same value?

File: LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift (hunk 1 above)
Comment (marinofelipe) [reply LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift]: It is already set as private in the protocol :). As `get` only.

File: LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift (hunk 2 above)
Comment (marinofelipe) [reply LivrCommonCustomRules/LivrCommonCustomRules/CustomRules/ExtraRules.swift]: Nice catch. I did wrongly conform to PreDefinedRule, which is not the case. Fixed already.

**Pull request merged**